### DISCUSSION TITLE
Homework (Sliding Number)

### DISCUSSION THREAD

**Discussion by MuhammadSulaiman001** [MuhammadSulaiman001/opengl-lab#14] [Announcements]
Hi everybody, [here](https://github.com/MuhammadSulaiman001/opengl-lab/tree/master/OpenGLab/Homework.Puzzle.SlidingNumber) you can find the homework specifications.

Please let me know in this discussion if you have any question.

**Comment by MuhammadSulaiman001** [comment]
Use VPN (ex. [Proton](https://protonvpn.com/download)) to be able to show images on GitHub.

**Comment by DaniaNouro** [comment]
Hello . I have a question. In order to achieve the sliding puzzel interface, I want to draw a large square and put an image on it, then inside it are 9 squares, and each square contains an image. This is the starting point, or is there another way to start?

**Reply by MuhammadSulaiman001** [reply]
Hi @DaniaNouro, answering your question is kind of cheating😅, anyway, the approach you've suggested is kinda intuitive! you can go with it.

**Reply by DaniaNouro** [reply]
Let's consider it a help 😁
 Thanks.
